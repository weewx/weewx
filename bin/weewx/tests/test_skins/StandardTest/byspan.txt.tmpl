Current time is $current.dateTime ($current.dateTime.raw)

Hourly Top Temperatures in Last Day
#for $_hour in $day(days_ago=1).hours
$_hour.start.format("%a %m/%d %H:%M")-$_hour.end.format("%H:%M"): $_hour.outTemp.max at $_hour.outTemp.maxtime
#end for

Daily Top Temperatures in Last Week
#for $_day in $week(weeks_ago=1).days
$_day.dateTime.format("%a %m/%d"): $_day.outTemp.max at $_day.outTemp.maxtime
#end for

Daily Top Temperatures in Last Month
#for $_day in $month(months_ago=1).days
$_day.dateTime.format("%a %m/%d"): $_day.outTemp.max at $_day.outTemp.maxtime
#end for

Monthly Top Temperatures this Year
#for $_month in $year.months
$_month.dateTime.format("%b %Y"): $_month.outTemp.max at $_month.outTemp.maxtime
#end for

3 hour averages over yesterday
#for $span in $yesterday.spans(interval=10800)
$span.dateTime.format("%m/%d %H:%M")-$span.end.format("%H:%M"): $span.outTemp.avg
#end for

All temperature records for yesterday
#for $record in $yesterday.records
$record.dateTime.format("%m/%d %H:%M"): $record.outTemp
#end for
