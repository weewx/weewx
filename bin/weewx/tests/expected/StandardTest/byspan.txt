Current time is 03-Sep-2010 11:00 (1283536800)

Hourly Top Temperatures in Last Day
Thu 09/02 00:00-01:00: 47.4F at 00:30
Thu 09/02 01:00-02:00: 43.7F at 01:30
Thu 09/02 02:00-03:00: 41.1F at 02:30
Thu 09/02 03:00-04:00: 39.7F at 03:30
Thu 09/02 04:00-05:00: 40.2F at 05:00
Thu 09/02 05:00-06:00: 42.2F at 06:00
Thu 09/02 06:00-07:00: 45.3F at 07:00
Thu 09/02 07:00-08:00: 49.4F at 08:00
Thu 09/02 08:00-09:00: 54.2F at 09:00
Thu 09/02 09:00-10:00: 59.4F at 10:00
Thu 09/02 10:00-11:00: 64.5F at 11:00
Thu 09/02 11:00-12:00: 69.3F at 12:00
Thu 09/02 12:00-13:00: 73.4F at 13:00
Thu 09/02 13:00-14:00: 76.6F at 14:00
Thu 09/02 14:00-15:00: 78.6F at 15:00
Thu 09/02 15:00-16:00: 79.2F at 16:00
Thu 09/02 16:00-17:00: 79.0F at 16:30
Thu 09/02 17:00-18:00: 77.7F at 17:30
Thu 09/02 18:00-19:00: 75.0F at 18:30
Thu 09/02 19:00-20:00: 71.3F at 19:30
Thu 09/02 20:00-21:00: 66.8F at 20:30
Thu 09/02 21:00-22:00: 61.7F at 21:30
Thu 09/02 22:00-23:00: 56.5F at 22:30
Thu 09/02 23:00-00:00: 51.4F at 23:30

Daily Top Temperatures in Last Week
Sun 08/22: 85.5F at 16:00
Mon 08/23: 84.9F at 16:00
Tue 08/24: 84.4F at 16:00
Wed 08/25: 83.9F at 16:00
Thu 08/26: 83.3F at 16:00
Fri 08/27: 82.7F at 16:00
Sat 08/28: 82.2F at 16:00

Daily Top Temperatures in Last Month
Sun 08/01: 94.7F at 16:00
Mon 08/02: 94.4F at 16:00
Tue 08/03: 94.0F at 16:00
Wed 08/04: 93.7F at 16:00
Thu 08/05: 93.3F at 16:00
Fri 08/06: 92.9F at 16:00
Sat 08/07: 92.5F at 16:00
Sun 08/08: 92.1F at 16:00
Mon 08/09: 91.7F at 16:00
Tue 08/10: 91.3F at 16:00
Wed 08/11: 90.8F at 16:00
Thu 08/12: 90.4F at 16:00
Fri 08/13: 89.9F at 16:00
Sat 08/14: 89.5F at 16:00
Sun 08/15: 89.0F at 16:00
Mon 08/16: 88.5F at 16:00
Tue 08/17: 88.0F at 16:00
Wed 08/18: 87.5F at 16:00
Thu 08/19: 87.0F at 16:00
Fri 08/20: 86.5F at 16:00
Sat 08/21: 86.0F at 16:00
Sun 08/22: 85.5F at 16:00
Mon 08/23: 84.9F at 16:00
Tue 08/24: 84.4F at 16:00
Wed 08/25: 83.9F at 16:00
Thu 08/26: 83.3F at 16:00
Fri 08/27: 82.7F at 16:00
Sat 08/28: 82.2F at 16:00
Sun 08/29: 81.6F at 16:00
Mon 08/30: 81.0F at 16:00
Tue 08/31: 80.4F at 16:00

Monthly Top Temperatures this Year
Jan 2010: 25.4F at 31-Jan-2010 15:00
Feb 2010: 38.7F at 28-Feb-2010 15:00
Mar 2010: 58.9F at 31-Mar-2010 16:00
Apr 2010: 78.8F at 30-Apr-2010 16:00
May 2010: 94.1F at 31-May-2010 16:00
Jun 2010: 100.0F at 30-Jun-2010 16:00
Jul 2010: 100.0F at 02-Jul-2010 16:00
Aug 2010: 94.7F at 01-Aug-2010 16:00
Sep 2010: 79.8F at 01-Sep-2010 16:00
Oct 2010:    N/A at    N/A
Nov 2010:    N/A at    N/A
Dec 2010:    N/A at    N/A

3 hour averages over yesterday
09/02 00:00-03:00: 43.4F
09/02 03:00-06:00: 40.4F
09/02 06:00-09:00: 48.6F
09/02 09:00-12:00: 63.2F
09/02 12:00-15:00: 75.5F
09/02 15:00-18:00: 78.3F
09/02 18:00-21:00: 70.0F
09/02 21:00-00:00: 55.2F

All temperature records for yesterday
09/02 00:30: 47.4F
09/02 01:00: 45.5F
09/02 01:30: 43.7F
09/02 02:00: 42.3F
09/02 02:30: 41.1F
09/02 03:00: 40.2F
09/02 03:30: 39.7F
09/02 04:00: 39.5F
09/02 04:30: 39.7F
09/02 05:00: 40.2F
09/02 05:30: 41.0F
09/02 06:00: 42.2F
09/02 06:30: 43.6F
09/02 07:00: 45.3F
09/02 07:30: 47.3F
09/02 08:00: 49.4F
09/02 08:30: 51.8F
09/02 09:00: 54.2F
09/02 09:30: 56.8F
09/02 10:00: 59.4F
09/02 10:30: 62.0F
09/02 11:00: 64.5F
09/02 11:30: 67.0F
09/02 12:00: 69.3F
09/02 12:30: 71.5F
09/02 13:00: 73.4F
09/02 13:30: 75.2F
09/02 14:00: 76.6F
09/02 14:30: 77.7F
09/02 15:00: 78.6F
09/02 15:30: 79.1F
09/02 16:00: 79.2F
09/02 16:30: 79.0F
09/02 17:00: 78.5F
09/02 17:30: 77.7F
09/02 18:00: 76.5F
09/02 18:30: 75.0F
09/02 19:00: 73.3F
09/02 19:30: 71.3F
09/02 20:00: 69.1F
09/02 20:30: 66.8F
09/02 21:00: 64.3F
09/02 21:30: 61.7F
09/02 22:00: 59.1F
09/02 22:30: 56.5F
09/02 23:00: 53.9F
09/02 23:30: 51.4F
09/03 00:00: 49.0F
